contracts/protocore/SelfProtocore.sol

```diff
@@ -79,7 +79,7 @@ contract SelfProtocore is MasterCopyNonUpgradable, GenesisSelfProtocore, Protoco
             genesisAuxiliaryTargetBlockNumber
         );
 
-        ValidatorSet.setupValidatorSet();
+        ValidatorSet.setupValidatorSet(openKernelHeight);
```

Comment:
As discussed today during the standup, iniital value for `activeHeight` must be one greater than the open kernel height.
So, it will be `ValidatorSet.setupValidatorSet(openKernelHeight.add(1))`

Reply:
Done.